app/components/course/CourseCard.tsx

```diff
@@ -18,14 +33,43 @@ const CourseCard = ({ title, courseCode, tag }: CourseCardProps) => {
             <p className="text-sm text-gray-600">{courseCode}</p>
           </header>
           <div className="flex gap-x-1 flex-wrap mt-2">
-            {tag?.map((t) => (
-              <p className="tag tag-primary" key={t}>
-                {t}
+            {properties.CI && (
+              <p className="tag tag-primary" key="CI">
+                Communication Intensive
+              </p>
+            )}
+            {properties.HI && (
+              <p className="tag tag-primary" key="HI">
+                Hass Inquiry
               </p>
```

Comment:
Doesn't need a key. Key is only for React to know what to update when using map function

Reply:
fixed